lib/bundles/inspec-compliance/api.rb

```diff
@@ -251,11 +251,55 @@ def self.server_version_from_config(config)
     end
 
     def self.determine_server_type(url, insecure)
-      if Compliance::HTTP.get(url + '/compliance/version', nil, insecure).code == '401'
-        :automate
-      elsif Compliance::HTTP.get(url + '/api/version', nil, insecure).code == '200'
-        :compliance
+      return :automate if target_is_automate_server?(url, insecure)
```

Comment:
Given that this is a pretty short method now that you've broken things out, I think it's totally fine to leave this as an `if...elsif` structure. No short-circuiting needed. In fact, I think it reads cleaner:

```ruby
if target_is_automate_server?(url, insecure)
  :automate
elsif target_is_compliance_server?(url, insecure)
  :compliance
else
  Inspec::Log.debug('Could not determine server type using known endpoints')
  nil
end
```

Reply:
I agree, will fix.